site/content/en/docs/tutorials/ebpf_tools_in_minikube.md

```diff
@@ -13,29 +13,21 @@ eBPF tools are performance tools used for observing the Linux kernel.
 These tools can be used to monitor your Kubernetes application in minikube.
 This tutorial will cover how to set up your minikube cluster so that you can run eBPF tools from a Docker container within minikube.
 
+
+## Requirment
+- x86 (currently bcc image does not support arm64)
 ## Prerequisites
 
 - Latest minikube binary
```

Comment:
We can probably update this to the version after the PR was merged to add eBPF to the ISO